cloudflare_worker/wrangler.example.toml

```diff
@@ -12,17 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name = "sxg"
-type = "rust"
-workers_dev = true
+name = 'sxg'
+type = 'rust'
+account_id = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+zone_id = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+routes = ['my_domain.com/*']
 
-zone_id = "XXXXX"
-account_id = "XXXXXX"
-routes = [
-  "my_domain.com/*",
-]
-kv_namespaces = [
-  # Please use the command "wrangler kv:namespace create OCSP" to create the id of kv namespace.
-  # The binding must be "OCSP".
-  { binding = "OCSP", id = "XXXXXX" },
-]
+[[kv_namespaces]]
+binding = 'OCSP'
+id = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+
+[vars]
+SXG_CONFIG = '''
+---
```

Comment:
Are you sure this `---` line is necessary?

Reply:
Deleted.